xcode/Subconscious/Shared/Components/BacklinksView.swift

```diff
@@ -10,7 +10,7 @@ import SwiftUI
 struct BacklinksView: View {
     var backlinks: [EntryStub]
     var onRequestDetail: (EntryLink) -> Void
-    var onLink: (_ context: Slashlink, SubSlashlinkLink) -> Void
+    var onLink: (_ context: EntryStub, SubSlashlinkLink) -> Void
```

Comment:
I'm generally in favor of passing as little information as possible to callbacks. A few open-ended thoughts...

IIRC there are a few places now where we need to pass a did with the slashlink. This feels like a hint that `slashlink` is not giving us everything we need to do addressing in every case.

We have the `Link` type which is an identifier made up of did + slug. If this is an option here, I would rather use it. However, this does lose petname information, which may be useful.

It pains me to say it but we may want to introduce an `Address` type that contains a DID plus slashlink, or perhaps a `Did`, `Petname?` and `Slug`.

Thoughts?

Reply:
Yep, I agree that the triple of `DID` + `Slashlink` or `DID` + `Petname?` + `Slug` deserves a type. I like the sound of `Address`, I'll explore that path.

Reply:
I ended up going with a more specific name: `ResolvedAddress`. We use the term address all over the codebase to refer to a `Slashlink` and from my POV a Slashlink is the most common kind of address in the app. `ResolvedAddress` is an attempt to carve out a more specific role.